site/src/main/java/uk/nhs/digital/ps/components/DatasetComponent.java

```diff
@@ -22,7 +23,7 @@ public void doBeforeRender(final HstRequest request, final HstResponse response)
         super.doBeforeRender(request, response);
         final HstRequestContext ctx = request.getRequestContext();
         Dataset dataset = (Dataset) ctx.getContentBean();
-        Publication publication = getParentPublication(ctx, dataset);
+        Publication publication = dataset.getParentPublication();
```

Comment:
can we use the same approach as in Publication tpl. set only dataset, and from tpl use `dataset.parentPublication.getSelfLinkBean`

Reply:
Will change this.